<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dataset</class>
 <widget class="QWidget" name="Dataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="dataset_area">
     <item>
      <widget class="QTreeWidget" name="image_list_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item>
        <property name="text">
         <string>トレーニング用画像</string>
        </property>
        <property name="icon">
         <iconset resource="../../qrc/icon.qrc">
          <normaloff>:/icon/resources/base/fonts/fontawesome/dumbbell_darkGray.png</normaloff>:/icon/resources/base/fonts/fontawesome/dumbbell_darkGray.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>良品</string>
         </property>
         <property name="icon">
          <iconset resource="../../qrc/icon.qrc">
           <normaloff>:/icon/resources/base/fonts/fontawesome/small_check-circle_correctGreen.png</normaloff>:/icon/resources/base/fonts/fontawesome/small_check-circle_correctGreen.png</iconset>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>性能評価用画像</string>
        </property>
        <property name="icon">
         <iconset resource="../../qrc/icon.qrc">
          <normaloff>:/icon/resources/base/fonts/fontawesome/gear-check_darkGray.png</normaloff>:/icon/resources/base/fonts/fontawesome/gear-check_darkGray.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>良品</string>
         </property>
         <property name="icon">
          <iconset resource="../../qrc/icon.qrc">
           <normaloff>:/icon/resources/base/fonts/fontawesome/small_check-circle_correctGreen.png</normaloff>:/icon/resources/base/fonts/fontawesome/small_check-circle_correctGreen.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不良品</string>
         </property>
         <property name="icon">
          <iconset resource="../../qrc/icon.qrc">
           <normaloff>:/icon/resources/base/fonts/fontawesome/times-circle_errorRed.png</normaloff>:/icon/resources/base/fonts/fontawesome/times-circle_errorRed.png</iconset>
         </property>
        </item>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="images_group_box">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="images_scroll_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="images_scroll_area_widget_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>554</width>
             <height>334</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
</string>
           </property>
           <layout class="QGridLayout" name="images_grid_area"/>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="actions_area">
          <item>
           <widget class="QPushButton" name="delete_images_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferDefault</stylestrategy>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>削除</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="actions_area_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="number_of_images_label">
            <property name="text">
             <string>0枚</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="camera_and_images_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 0 10px;</string>
            </property>
            <property name="text">
             <string>追加</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="training_area">
     <item>
      <spacer name="training_area_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="latest_training_date_label">
       <property name="styleSheet">
        <string notr="true">margin-right: 4px</string>
       </property>
       <property name="text">
        <string>前回のトレーニング 2019/04/24</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="train_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>トレーニング</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="select_images_action">
   <property name="icon">
    <iconset resource="../../qrc/icon.qrc">
     <normaloff>:/icon/resources/base/fonts/fontawesome/image_black.png</normaloff>:/icon/resources/base/fonts/fontawesome/image_black.png</iconset>
   </property>
   <property name="text">
    <string>既存の画像 </string>
   </property>
   <property name="toolTip">
    <string>既存の画像</string>
   </property>
  </action>
  <action name="camera_action">
   <property name="icon">
    <iconset resource="../../qrc/icon.qrc">
     <normaloff>:/icon/resources/base/fonts/fontawesome/camera_black.png</normaloff>:/icon/resources/base/fonts/fontawesome/camera_black.png</iconset>
   </property>
   <property name="text">
    <string>カメラで撮影</string>
   </property>
   <property name="toolTip">
    <string>カメラで撮影</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../qrc/icon.qrc"/>
 </resources>
 <connections/>
</ui>
